README.md

```diff
@@ -27,13 +27,11 @@ This cookiecutter:
 
 ### Prerequisities
 
-Cookiecutter version 1.7.x installed.
-
-You can satify this requirement with ```pip install cookiecutter```
+You just need to have pipenv installed.
```

Comment:
I would add a link to the Pipenv official repository

Reply:
done!